icons/folder-gitlab-open.svg

```diff
@@ -1,6 +1,6 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
   <path d="M28.96692,12H9.44152a2,2,0,0,0-1.89737,1.36754L4,24V10H28a2,2,0,0,0-2-2H15.1241a2,2,0,0,1-1.28038-.46357L12.5563,6.46357A2,2,0,0,0,11.27592,6H4A2,2,0,0,0,2,8V24a2,2,0,0,0,2,2H26l4.80523-11.21213A2,2,0,0,0,28.96692,12Z" style="fill: #757575"/>
-  <g>
+  <g mit-no-recolor="true">
```

Comment:
I assume that's just an acronym for "Material Icon Theme."

Reply:
@Repiteo that makes sense 🙂 Then we can keep it like that.

Reply:
Exactly, as Repiteo mentioned, it's an acronym for "Material Icon Theme". I didn't want to go with `no-recolor` without a prefix, just in case `no-recolor` becomes a valid attribute in the SVG specification in the future (highly improbable, but who knows).